install/build_from_source.md

```diff
@@ -56,7 +56,57 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 ## ROOT, Python and PyROOT
 
-> TODO: explain what build options are relevant, what they do, Python2/Python3, etc.
+PyROOT, the set of Python bindings of ROOT, changed its structure and build/installation process in v6.22. In the following the main aspects are summarized for both after and before v6.22.
+
+### After v6.22
+
+The main feature introduced in v6.22 concerning the PyROOT build system is the possibility to build for both Python3 and Python2 (MultiPython), available only if the version of CMake used to build is >= 3.14. In the following, build and installation processes both with CMake >= 3.14 and < 3.14 are described. 
+
+#### CMake >= 3.14
+
+In this case, PyROOT libraries are built by default with both Python3 and Python2. For each Python version X.Y used to build PyROOT (e.g. 3.8, 2.7, etc.) the following libraries (containing the C++ extensions) will appear both in the build directory and in the installation directory:
+- `libROOTPythonizationsX_Y.so`
+- `libcppyX_Y.so`
+- `libcppyy_backendX_Y.so`
+
```

Comment:
What about `TPython`? I think we should mention here that in the case of `libROOTTPython.so`, containing the `TPython` functionality, it will be generated just for the highest Python version with which ROOT is built.